aboutus.css

```diff
@@ -0,0 +1,60 @@
+*{
+    margin: 0px;
+    margin-top: -3px;
```

Comment:
unnecessary code.
no change will happen if you delete it or keep it
( delete it )